Forgot.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Actions } from 'react-native-router-flux';
+
+export default class Forgot extends Component {
+	render() {
```

Comment:
These are all indented a bunch. I believe you can set your indentation preferences in sublime. I like using Tabs->Spaces and setting the indentation of a tab to equal 2 spaces